### PR TITLE
[INLONG-6542][Dashboard] Optimize stream execution workflow and execution log

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/6542

### CODE REVIEW

**Review by leezng — APPROVED** [review]

File: inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx [hunk 1]
```diff
@@ -178,11 +184,14 @@ const Comp = ({ inlongGroupId, readonly, mqType }: Props, ref) => {
...
             </Button>
-            {record?.status && (record?.status === 120 || record?.status === 130) && (
-              <Button type="link" onClick={() => onWorkflow(record)}>
-                {t('meta.Stream.ExecuteWorkflow')}
-              </Button>
-            )}
+            {record?.status &&
+              ((groupStatus === 130 && record?.status === 100) ||
+                groupStatus === 120 ||
+                groupStatus === 130) && (
```
Comment (healchow) [inline inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx:190]: Could you please move those two lines into one?

Comment (bluewang) [reply inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx:190]: An error occurs when merging：
image

File: inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx [hunk 2]
```diff
@@ -72,6 +74,10 @@ const Comp = ({ inlongGroupId, readonly, mqType }: Props, ref) => {
     },
   );
 
+  const groupInfo = useRequest(`/group/get/${inlongGroupId}`, {
```
Comment (leezng) [inline inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx:77]: remove `const groupInfo =` ?

**Review by healchow — APPROVED** [review]

File: inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx [hunk 3]
```diff
@@ -178,11 +184,14 @@ const Comp = ({ inlongGroupId, readonly, mqType }: Props, ref) => {
             <Button type="link" onClick={() => onDelete(record)}>
               {t('basic.Delete')}
             </Button>
-            {record?.status && (record?.status === 120 || record?.status === 130) && (
-              <Button type="link" onClick={() => onWorkflow(record)}>
-                {t('meta.Stream.ExecuteWorkflow')}
-              </Button>
-            )}
+            {record?.status &&
+              ((groupStatus === 130 && record?.status === 100) ||
```
Comment (leezng) [inline inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx:188]: Are the two conditions `groupStatus === 130 && record?.status === 100` and `groupStatus === 130` repeated, and only need `groupStatus === 130`?

**Review by leezng — APPROVED** [review]

**Pull request merged**